cargo-near/src/lib.rs

```diff
@@ -47,6 +47,9 @@ pub struct AbiCommand {
     /// Path to the `Cargo.toml` of the contract to build
     #[clap(long, parse(from_os_str), value_name = "PATH")]
     pub manifest_path: Option<PathBuf>,
+    /// Do not prettify ABI (will generate minified JSON)
+    #[clap(long)]
+    pub no_pretty: bool,
```

Comment:
idk, this doesn't seem very intuitive to me. what do you think of `--minify` or `--compact`?

Reply:
Unclear, seems opinionated. with `--compact` that is ambiguous whether it's referring to the ABI or the contract. Another option to throw in the ring is `--compact-abi`? `--no-pretty` still seems fine.

Or we could just remove altogether and if someone needs to compact it, they just use some external tool to compact it themselves, or we just wait until someone asks for this feature.

Reply:
Googled `--no-pretty` and there are a couple of well-known projects that use it, but overall it is indeed pretty rare. I am just going with `--compact-abi`. Seems the least opinionated, but we can always change while in early semver it if there is a strong reason for it.